languages/scala/dockerfiles/scala-3.3.5.Dockerfile

```
@@ -0,0 +1,47 @@
+RUN chmod +x /codecrafters-precompile.sh
+
+# syntax=docker/dockerfile:1.7-labs
+FROM maven:3.9.9-eclipse-temurin-17-alpine
+
+# Ensures the container is re-built if dependency files change
+ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="build.sbt"
+
+WORKDIR /app
+
+# .git & README.md are unique per-repository. We ignore them on first copy to prevent cache misses
+COPY --exclude=.git --exclude=README.md . /app
```

Hadolint (2.12.0)

[error] invalid flag: --exclude

(DL1000)